test/stdcpp/Makefile

```diff
@@ -0,0 +1,18 @@
+include ../common.mak
+
+TESTS:=array
+
+.PHONY: all clean
+all: $(addprefix $(ROOT)/,$(addsuffix .done,$(TESTS)))
+
+$(ROOT)/test_stdcpp.done: $(ROOT)/%.done : $(ROOT)/%
+	@echo Testing $*
+	$(QUIET)$(TIMELIMIT)$(ROOT)/$* $(RUN_ARGS)
+	@touch $@
+
+$(ROOT)/%: $(SRC)/%.d $(SRC)/%.cpp
+	$(QUIET)$(CC) $(CFLAGS) -c -o $(ROOT)/%_cpp.o $(SRC)/%.cpp
```

Comment:
use `$*` instead of `%` inside the rules.

Reply:
The dated C++ compiler installed in my VM doesn't create the target directory, so you have to prefix this with
```
	mkdir -p $(dir $@)
```